frontend/src/constants/paths.tsx

```diff
@@ -0,0 +1,7 @@
+export  const PATH_FOR_HOME_VIEW : string = "/";
+export const PATH_FOR_LOGIN_VIEW : string = "/login";
+export  const PATH_FOR_TOURNAMENTS_SELECTION_VIEW : string = "/tournaments-selection";
+export  const PATH_FOR_CREATING_NEW_TOURNAMENT_VIEW : string = "/new-tournament";
+export  const PATH_FOR_TOURNAMENT_REGISTRATIONS_VIEW : string = "/tournament-registration";
```

Comment:
Pamiętasz wymaganie a porpos /tournament-registration? Tam był path podany inny :) ? Może to być zawsze w kolejnym PR.

Reply:
ok